Libraries/Opc.Ua.Client/Subscription.cs

```diff
@@ -724,6 +799,29 @@ public void Create()
 
             ChangesCompleted();
         }
+        private void ManageMessageWorkerSemaphore()
```

Comment:
I *think* the private methods are in a seperate region?

Reply:
I'll have a look again